bbc/simorgh
# WSTEAM1-380: Display title and description on Live Page

Resolves JIRA [WSTEAM1-380](https://jira.dev.bbc.co.uk/browse/WSTEAM1-380)

Overall changes
======
Show title and description supplied by pageData on the Live Page.

Code changes
======
- Use destructing to extract title and description from the pageData and render values in `<Heading>` and `<Text>` tags.
- Added a comment above the `<Text>` element to remind us that this has not been subject to Screen Reader UX/ UX review.
- Update initial Jest tests and update snapshot so that they pass.

Notes
======
- I have not updated the `MetadataContainer` values (in case these use SEO values/ are outside the scope of this ticket)
- I have made the title and description props optional. I do not know whether it is possible to publish a live page with one/ both of these values missing
- Is the `<Text>` comment necessary?
- I have not tested this against the pageData title/ description value changing (see A/C in ticket). But I assume this would update.

Testing
======
1. Pull down PR, cd into ws-nextjs-app and launch the local server
2. Visit http://localhost:7081/pidgin/new_live/c7p765ynk9qt
3. Check that the title and description showing matches the pageData

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
